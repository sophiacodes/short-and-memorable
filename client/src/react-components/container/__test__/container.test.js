import React from 'react';
import Container from '../';
import renderer from 'react-test-renderer';

it('renders <Container> correctly', () => {
  const tree = renderer
    .create(<Container><p>P element inside container test</p></Container>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});