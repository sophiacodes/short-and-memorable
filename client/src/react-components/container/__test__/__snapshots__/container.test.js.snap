// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders <Container> correctly 1`] = `
<section
  className="container"
>
  <p>
    P element inside container test
  </p>
</section>
`;
